roundedimageview/src/main/java/com/makeramen/roundedimageview/PicassoRoundedTransformationBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.makeramen.roundedimageview;
+
+import android.graphics.Bitmap;
+
+import com.squareup.picasso.Transformation;
+
+public class PicassoRoundedTransformationBuilder {
+
+  private RoundedTransformationBuilder mBuilder;
+
+  public PicassoRoundedTransformationBuilder(final RoundedTransformationBuilder builder) {
+    mBuilder = builder;
```

Comment:
should this be a copy to avoid extra mutation of the original builder after calling .picasso()?